main/options/options.h

```diff
@@ -211,6 +211,7 @@ struct Options {
     bool lspDocumentSymbolEnabled = false;
     bool lspDocumentFormatRubyfmtEnabled = false;
     bool lspSignatureHelpEnabled = false;
+    bool lspGoToImplementationEnabled = true;
```

Comment:
```suggestion
    bool lspGoToImplementationEnabled = false;
```

Reply:
I had it set to `true` for the ease of testing. Change it to `false` as it should be.